### PR TITLE
Added `database_id` deprecation notice

### PR DESCRIPTION
**Pull request by ezilber-akamai**
## 📝 Description

Added `database_id` deprecation notice

### CODE REVIEW

**Review by yec-akamai — APPROVED** [review]

**Comment by zliang-akamai** [comment]
Can you also update example?
https://github.com/linode/terraform-provider-linode/pull/859/files#diff-90358e8869220924dcbe6f87796e1563542df2a3476fc100b67d98e328b0c1c7R17-R21

**Comment by ezilber-akamai** [comment]
> Can you also update example? https://github.com/linode/terraform-provider-linode/pull/859/files#diff-90358e8869220924dcbe6f87796e1563542df2a3476fc100b67d98e328b0c1c7R17-R21

Done!

File: linode/databasepostgresql/framework_datasource_schema.go [hunk 1]
```diff
@@ -11,6 +11,7 @@ import (
 
 var frameworkDatasourceSchema = schema.Schema{
 	Attributes: map[string]schema.Attribute{
+		// DEPRECATED: Use ID instead
```
Comment (zliang-akamai) [inline linode/databasepostgresql/framework_datasource_schema.go:14]: Can we move the note to the description of the attribute?

File: website/docs/d/database_postgresql.html.markdown [hunk 2]
```diff
@@ -72,6 +72,8 @@ The `linode_database_postgresql` data source exports the following attributes:
 
 * `version` - The Managed Database engine version. (e.g. `v8.0.26`)
 
+* `id` - The ID of the PostgreSQL database.
+
```
[the remark this answers is not in the document]
Comment (ezilber-akamai) [reply website/docs/d/database_postgresql.html.markdown]: Done!

File: linode/databasepostgresql/framework_datasource_schema.go (hunk 1 above)
Comment (ezilber-akamai) [reply linode/databasepostgresql/framework_datasource_schema.go]: Done!

**Review by zliang-akamai — APPROVED** [review]
Nice work!

**Review by lgarber-akamai — APPROVED** [review]

**Pull request merged**